app/components/FileRow.js

```diff
@@ -258,13 +255,16 @@ export default class FileRow extends Component {
   }
 
   render() {
-    return (
-      <Table.Row>
-        {this.generateSelectCell()}
-        {this.generateDetailsCell()}
-        {this.generateStartTimeCell()}
-        {this.generateOptionsCell()}
-      </Table.Row>
-    );
+    if(fs.existsSync(this.props.file.fullPath)){
```

Comment:
Just my two cents:
* I think it's better to actually remove the file from the entire file list rather than check for file existence before rendering. The rendering should expect the file to exist and just render the component rather than worry about it not existing.
* It's probably a good idea to use [`shell.moveItemToTrash`](https://www.electronjs.org/docs/api/shell#shellmoveitemtotrashfullpath-deleteonfail) instead of `fs.unlink` since that way the files can be recovered from the Recycle bin if they so wish.

Reply:
Ok, I'll try that out

Reply:
I did it